### PR TITLE
[CHECKOUTUI-106] Allow customization of InStore checkout files via admin when using Checkout UI Custom app

### PR DESCRIPTION
**Pull request by AnnaChiu95**
#### What does this PR do? 
When [vtex.checkout-ui-custom](https://github.com/vtex-apps/checkout-ui-custom) or [vtex.checkout-ui-settings](https://github.com/vtex-apps/checkout-ui-settings) is installed in a workspace, the checkout files saved in the admin will no longer apply. This PR enables users to edit and apply InStore checkout files (checkout-instore-custom.css and checkout-instore-custom.js) in the admin for workspaces with `vtex.checkout-ui-custom` installed.

#### How to test it? 
Test in workspace [annacheckout](https://annacheckout--sandboxusdev.myvtex.com/admin).

Here's a [cart link](https://annacheckout--sandboxusdev.myvtex.com/checkout/cart/add/?sku=2&qty=1&seller=1&sc=1). Go to Inspect > Sources and view the checkout files. For this workspace, I mimicked the scenario using a custom version of `vtex.checkout-ui-settings` and enabled users to edit checkout6-custom.css via the admin while the checkout6-custom.js content comes from the checkout-ui-settings app.

_checkout6-custom.css using [admin content](https://annacheckout--sandboxusdev.myvtex.com/admin/portal/#/sites/default/code/files/checkout6-custom.css)_
Screen Shot 2022-09-02 at 5 27 10 PM

_checkout6-custom.js not using [admin content](https://annacheckout--sandboxusdev.myvtex.com/admin/portal/#/sites/default/code/files/checkout6-custom.js)_
Screen Shot 2022-09-02 at 5 19 36 PM

### CODE REVIEW

File: node/clients/portalFiles.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+export default class PortalFiles extends ExternalClient{
+    constructor(ctx: IOContext, options?: InstanceOptions){
+        super(baseURL(ctx),ctx, {
+            ...options,
+            headers: {
+                ...(options?.headers ?? {}),
+                'X-Vtex-Use-Https': 'true',
+                VtexIdclientAutCookie: ctx.adminUserAuthToken ?? ctx.authToken,
```
Comment (polishq) [inline node/clients/portalFiles.ts:20]: I would use `ctx.authToken` in all cases, as we have control over the app's permissions but the user's can vary

**Review by polishq — CHANGES_REQUESTED** [review]

File: node/clients/portalFiles.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+interface File {
+    path: string
+    text: string
+}
+
+const baseURL = ({ workspace, account }: IOContext) => {
+    return `http://${workspace}--${account}.myvtex.com`
+}
+
+export default class PortalFiles extends ExternalClient{
```
[the remark this answers is not in the document]
Comment (AnnaChiu95) [reply node/clients/portalFiles.ts]: Ok, I switched to `JanusClient`. I was previously getting 403 errors so I thought it had something to do with the client but now I see that it was really an issue with `ctx.authToken`

File: node/clients/portalFiles.ts (hunk 1 above)
Comment (AnnaChiu95) [reply node/clients/portalFiles.ts]: Done.

**Review by polishq — APPROVED** [review]
LGTM

**Pull request merged**